packages/serverless-contracts/package.json

```diff
@@ -13,16 +13,15 @@
     "Typescript"
   ],
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "directory": "dist",
+    "linkDirectory": true
```

Comment:
Let's wait for https://github.com/swarmion/swarmion/pull/608 to have linkDirectory: true enabled by default and remove this line

Reply:
done !